### DISCUSSION TITLE
Generating reports in Docker - error

### DISCUSSION THREAD

**Discussion by sterankin** [artilleryio/artillery#1483] [Artillery]
I am running a Docker command using the Docker artillery image, i want to generate an HTML report.  But it is failing to find the report.  The tests run ok, and the json output is correct, however running the report command fails to find the report.json (even though the actual path is provided to the json report)

Version info:
```
Artillery Core: 2.0.0-18
Node.js: v17.8.0
```

Running this command: 
```
docker run  --rm  --volume ${PWD}/testing-apis/artillery:/usr/src/app artilleryio/artillery:latest run /usr/src/app/artillery-config.yml --output /usr/src/app/result.json && artillery report --output /usr/src/app/report.html /usr/src/app/result.json
```
OR

```
docker run  --rm  --volume ${PWD}/testing-apis/artillery:/usr/src/app artilleryio/artillery:latest run artillery-config.yml --output result.json && artillery report --output report.html result.json
```

I expected to see this happen:

HTML report would be generated.

Instead, this happened:

```
Log file: /usr/src/app/result.json
    Error: ENOENT: no such file or directory, open 'result.json'
    Code: ENOENT
Error: ENOENT: no such file or directory, open 'result.json'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at ReportCommand.run (/usr/local/lib/node_modules/artillery/lib/cmds/report.js:13:32)
    at ReportCommand._run (/usr/local/lib/node_modules/artillery/node_modules/@oclif/command/lib/command.js:43:31)
ENOENT
ENOENT: no such file or directory, open 'result.json'
```

**Comment by hassy** [comment]
`artillery report` invoked after `&&` is running on the host I would guess, and not in the Docker container

**Reply by sterankin** [reply]
Thanks @hassy.  Is there a way to ensure it gets run in the container?  Since there is no /bin/bash in the container I'm not sure commands can be passed?

I'm trying to run this in CI as part of a single docker command.

**Reply by sterankin** [reply]
I have also noticed that the Docker command does not fail when I use the `ensure` config setting in the yaml file.  If I set it to something low like 20ms, and my p95 is something like 1500, I would expect there to be a failure.

```
  ensure:
    p95: 2000
    maxErrorRate: 1
```

**Reply by sterankin** [reply]
I assume the Docker image does not have the ensure plugin installed?
